### DISCUSSION TITLE
Lava fabricator, just like it used to be

### DISCUSSION THREAD

**Discussion by SuperBrainAK** [InnovativeOnlineIndustries/Industrial-Foregoing#925] [Completed/Done]
Back in the day I was able to get buckets of lava for my mekanism crusher recipe, since MFR has been through its various redo's it has been lost. I would like it back, here is the wiki page of the old machine for reference: https://ftbwiki.org/Lava_Fabricator
The only thing I would change would maybe have it require a handfull of cobblestone to heat up into lava.

**Comment by WBgroup** [comment]
You can generate lava with a fluid laser now tho

**Comment by Buuz135** [comment]
Correct

**Comment by SuperBrainAK** [comment]
oh I see that now, I havent done anything with lasers yet in my base build, it would still be nice to have the dedicated machine instead of a laser drill in the nether.
